bin/main

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts 'Hello!'
```

Comment:
- Instead of `Hello!`, please use the message specified in the pathwright instructions here: 
![image](https://user-images.githubusercontent.com/2739245/91916498-ed569700-ec82-11ea-8338-9e37274da0c2.png)